build.gradle

```diff
@@ -240,7 +240,7 @@ checkstyleTest.enabled = false
 dependencies {
 
     def jacksonVersion = "2.13.4"
-    def jacksonDataBindVersion = "2.13.4"
+    def jacksonDataBindVersion = "2.13.4.2"
```

Comment:
Need this change in RCA as well - https://github.com/opensearch-project/performance-analyzer-rca/blob/main/build.gradle#L319

Reply:
Done in https://github.com/opensearch-project/performance-analyzer-rca/pull/243